.github/workflows/ci.yml

```diff
@@ -2,14 +2,13 @@ name: BentoML-CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
```

Comment:
the same question for this file, did you apply an auto-formatting program for this yml file? and could you make it a separate PR?